tests/test_utility.py

```diff
@@ -7,6 +7,18 @@
 from app.api import utility as util
 
 
+@pytest.mark.parametrize(
+    "url, expected_url",
+    [
+        ("https://publicnode.org", "https://publicnode.org/"),
+        ("https://publicnode.org/", "https://publicnode.org/"),
+    ],
+)
+def test_add_trailing_slash(url, expected_url):
+    """Test that a trailing slash is added to a URL if it does not already have one."""
+    assert util.add_trailing_slash(url) == expected_url
+
+
```

Comment:
that's cool, but I think this is a new test. Did we just forget this before? Or is this a new fix that we can link to an issue?

Reply:
Yeah, it got left behind essentially. This was a test suggested in your review of the last PR for this repo, https://github.com/neurobagel/federation-api/pull/30#discussion_r1399643369, I made the change but realized I forgot to push it to the branch before I merged it on GitHub 🙃